src/components/PantallaValidacion/BoardResolution/index.module.scss

```
@@ -0,0 +1,171 @@
+.container{
```

Aca falta un espacio
```suggestion
.container {
```